.github/workflows/release.yaml

```diff
@@ -8,7 +8,7 @@ name: Release Workflow
 on:
   push:
     tags:
-      - 'v.*'
+      - 'v*'
```

Comment:
The previous match was wrong, it would match on tags starting with `v.`. Read this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags.

Reply:
So difference b/w regexps and file globs?

Reply:
right